fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/CompoundPredicateOperator.java

```diff
@@ -90,7 +89,13 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(opType, type, Sets.newHashSet(this.getChildren()).hashCode());
+        int h = 0;
```

Comment:
is it better a hash function?

Reply:
it's a initialized value, we should keep same value every time call this function, and I think there is no need to use hash function, it also may have performance loss.